src/rgw/setup/rgw.py

```diff
@@ -117,6 +117,17 @@ def config(conf: MappedConf):
         os.environ['M0_TRACE_DIR'] = motr_trace_dir
         Log.info('Created motr trace directory : %s' % os.environ.get('M0_TRACE_DIR'))
 
+        # Create motr addb directory for collecting addb files
+        # in case admin user creation issue during mini-provisioner execution.
+        Log.info('Creating motr addb_files- directory for collecting addb files..')
+        motr_fid_key = const.MOTR_MY_FID % index
```

Comment:
In config phase for addb dir creation can we use admin fid value(const.MOTR_ADMIN_FID_KEY) instead of 'const.MOTR_MY_FID % index'.

Reply:
your suggestion works properly. Thanks